workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.ts

```diff
@@ -72,8 +72,14 @@ export class WBCGroupManagementPlugin implements GroupManagementPlugin {
   public getGroupUsers(request: GetGroupUsersRequest): Promise<GetGroupUsersResponse> {
     throw new Error('Method not implemented.');
   }
-  public addUserToGroup(request: AddUserToGroupRequest): Promise<AddUserToGroupResponse> {
-    throw new Error('Method not implemented.');
+  public async addUserToGroup(request: AddUserToGroupRequest): Promise<AddUserToGroupResponse> {
+    const { groupId, userId } = request;
+
+    // ToDo: Any remapping of following exceptions returned from _userManagementService.addUserToRole needs to be performed?
+    // IdpUnavailableError, PluginConfigurationError, UserNotFoundError, RoleNotFoundError
+
+    await this._userManagementService.addUserToRole(userId, groupId);
+    return { data: { added: true } };
```

Comment:
return userId and group Id as the data instead. In reference to previous slack discussions, added would never be set to true because errors would be thrown otherwise